contracts/pcv/IPCVDepositV2.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "./IPCVDeposit.sol";
+
+/// @title a PCV Deposit interface for Fei V2
+/// @author eswak
+interface IPCVDepositV2 is IPCVDeposit {
```

Comment:
Ultimately we should just add these methods to the original interface (but I know that would lead to many compilation errors so we should wait until ready to add all of these methods to existing contracts)